### DISCUSSION TITLE
[Question][lecture0.pdf]

### DISCUSSION THREAD

**Discussion by munimwasi** [prosyslab-classroom/cs348-information-security#652] [Q&A]
Name: Munim Hasan Wasi

In page 17 of the lecture0.pdf, the slide shows that the answer here is 17. But shouldn't it be 18? ![image](https://user-images.githubusercontent.com/29173998/223625249-1c109725-fcac-4b10-8dbd-3d4cb14b6bac.jpeg)

**Comment by KihongHeo** [comment]
Hi Munim.

Nice catch!
